README.md

```diff
@@ -58,6 +58,19 @@ npm i changelog-maker -g
 * `--all` or `-a`:     process all commits since beginning, instead of last tag.
 * `--help` or `-h`:    show usage and help.
 
+## Development
+
+You will need to generate a [personal access token](https://github.com/settings/tokens) with `public_repo` scope. Then create a config.json file:
```

Comment:
"You will need" shouldn't be the case - if you _run_ it the first time on a repo that has `PR-URL` metadata then it _should_ prompt you for GitHub login to make this itself. There really shouldn't be a need to manually mess with these files.

I'm still interested in why this didn't get created for you. Did you try running it manually against this repo? or nodejs/node? a bunch of nodejs repos have `PR-URL` in them as standard. Maybe it's just the test runner that doesn't make this so these instructions should say "before running tests you will need to run `node changelog-maker.js` to be prompted with a GitHub login to make an access token" .. then continue to explain what this token is and where it's located like you've done here. It just shouldn't be necessary to manually do this though.

Reply:
I can't confirm this because I did run it without any arguments and nothing happened. Maybe that's why?

Remember, say one wants to help with this repo, they clone it, do `npm i` and then `npm t` and voila, tests will fail, at least they did for me.

Reply:
ah ok, so I've just moved my config out of the way and tried to trigger the login - because a fetch from GitHub is only triggered by `PR-URL` metadata, just running `./changelog-maker.js` doesn't do it because there are no recent commits. But if you run `./changelog-maker.js -a` then it'll fetch all history and trigger a login because it'll get to commits with `PR-URL`.

So I'd suggest, for this section, to provide two options, the first one is to run `./changelog-maker.js -a` to trigger a login, which will save a config file, and the second option is the one you've laid out here. In fact, some people may prefer to use this second option rather than to have a mysterious program handle your github credentials (I've had at least one person want to do it manually in the past). So this information is helpful, but is complicated so should be documented as a secondary option.